bundle/libraries/libraries.go

```diff
@@ -111,5 +112,20 @@ func libPath(library *compute.Library) string {
 }
 
 func isLocalLibrary(library *compute.Library) bool {
-	return libPath(library) != ""
+	path := libPath(library)
+	if path == "" {
+		return false
+	}
+
+	return !isDbfsPath(path) && !isWorkspacePath(path)
```

Comment:
Another indicator of non-local is that it'll be an absolute path. We use this in the notebook path translation logic. We should combine the two into a single code path that determines if a path is local or not.